src/kvstore/raftex/RaftPart.h

```diff
@@ -190,6 +194,11 @@ class RaftPart : public std::enable_shared_from_this<RaftPart> {
      */
     void checkAndResetPeers(const std::vector<HostAddr>& peers);
 
+    /**
+     * Add listener into peers or remove from peers
```

Comment:
the comment is mismatch with the function name ?

Reply:
It will check if listener is same with expect value, and then add/remove if not match.